docs/pages/database-access/guides/aws-dynamodb.mdx

```diff
@@ -0,0 +1,265 @@
+```
+
+(!docs/pages/includes/add-role-to-user.mdx role="aws-dynamodb-access"!)
+
+## Step 3/4. Install the Teleport Database Service
+
+Create and EC2 instance to host the Teleport Database Service, and attach the
+`TeleportDatabaseService` to it. If you're hosting the service another way, you
+must provide AWS authentication to the host manually.
+
+<Admonition type="warning" title="non-standard AWS regions">
+For non-standard AWS regions such as AWS GovCloud (US) regions and AWS China
+regions, please set the corresponding region in the `AWS_REGION` environment
+variable or in the AWS credentials file so that the Application Service can use
+the correct STS endpoint.
+</Admonition>
+
+### Generate a token
+
+A join token is required to authorize a Teleport Database Service instance
+to join the cluster. Generate a short-lived join token and save the output of
+the command:
+
+```code
+$ tctl tokens add \
+    --type=db \
+    --db-name=aws-dynamodb
+```
+
+Use the token provided by the output of this command in the next step.
+
+### Install and start Teleport
+
+Install Teleport on the host where you will run the Teleport Application
+Service. See our [Installation](../../installation.mdx) page for options
+besides Linux servers.
+
+(!docs/pages/includes/install-linux.mdx!)
+
+Edit `/etc/teleport.yaml` to configure the database entry:
+
+```yaml
+version: v3
+teleport:
+  nodename: CHANGEME
+  data_dir: /var/lib/teleport
+  proxy_server: teleport.example.com
+  auth_token: (=presets.tokens.first=)
+  ca_pin:
+  - (=presets.ca_pin=)
+db_service:
+  enabled: "yes"
+  # Matchers for database resources created with "tctl create" command.
+  # For more information: https://goteleport.com/docs/database-access/guides/dynamic-registration/
+  resources:
+  - labels:
+      "*": "*"
+  # Lists statically registered databases proxied by this agent.
+  databases:
+    - name: "aws-dynamodb"
+      protocol: "dynamodb"
+      # optional uri, if uri is set then AWS region can be extracted from it
+      # or if AWS region is already set then the regions must match.
+      # uri: "dynamodb.us-east-1.amazonaws.com:443"
+      static_labels:
+        env: "dev"
+      aws:
+        region: "us-east-1"
+        account_id: "(=aws.aws_access_key=)"
+```
+
+Now start the Teleport Database Service:
+
+<Tabs>
+<TabItem label="Package Manager">
+
+```code
+$ sudo systemctl start teleport
+```
+
+</TabItem>
+<TabItem label="TAR archive">
+
+```code
+$ sudo teleport install systemd --output=/etc/systemd/system/teleport.service;
+$ sudo systemctl enable teleport
+$ sudo systemctl start teleport
+```
+
+</TabItem>
+</Tabs>
+
+## Step 4/4 Connect
+
+Once the Database Service has started and joined the cluster, you can start
+connecting to your DynamoDB database.
+
+Create a proxy tunnel:
+
+```code
+$ tsh proxy db --tunnel --port 8000 --db-user=ExampleTeleportDynamoDBRole dynamodb-name
+```
+
+You can test the connection to the database through the `aws` cli:
```

Comment:
You can also use NoSQL Workbench as described in my PR.

Please update "connect-your-client/gui-clients.mdx" to include a section for DynamoDB with NoSQL Workbench.

Reply:
I suggest including my example of using boto3 python here as well. They can use SDK through the proxy tunnel.

That way we cover:
1. aws cli access
2. GUI client access
3. programmatic access

Reply:
I've updated `gui-clients.mdx` and linked to it, and provided your example for programmatic access. Please take a look, and mark this as resolved if you deem it so.